packages/mui-material/src/AvatarGroup/AvatarGroup.d.ts

```diff
@@ -15,6 +17,14 @@ export interface AvatarGroupProps extends StandardProps<React.HTMLAttributes<HTM
    * Override or extend the styles applied to the component.
    */
   classes?: Partial<AvatarGroupClasses>;
+  /**
+   * The props used for targeting the additional avatar when max prop is set.
```

Comment:
`componentsProps` is more generic. You can use a description similar to ones found in other components, such as:
```js
 /**
  * The props used for each slot inside the AvatarGroup.
  * @default {}
  */
```

Reply:
done